ext/dom/html5_parser.c

```diff
@@ -181,6 +195,25 @@ static lexbor_libxml2_bridge_status lexbor_libxml2_bridge_convert(
                 }
 
                 lxml_attr->children = lxml_attr->last = lxml_text;
+                lxml_text->parent = (xmlNodePtr) lxml_attr;
+
+                if (attr->node.ns == LXB_NS_XMLNS) {
+                    if (strcmp((const char *) local_name, "xmlns") != 0) {
```

Comment:
Is this cast needed?

Reply:
Yes, to convert `const lxb_char_t *` (aka `const unsigned char *`) to `const char *`.